docs/releasing.md

```diff
@@ -33,13 +33,7 @@ Tada! The pull requests in this release now say the version they released in rig
 
 ### Finalize documentation
 
-Before cutting the release, do what you can to square away the documentation for the release. The documentation is in the `docs` folder right in the codebase and they'll be tagged and available for download with the code, so it's better to square it away before tagging and releasing rather than having errata not included with the release.
-
-### Node release
-
-+ Update the npm package by running `npm version <update_type>` where <update_type> is based on the selection above: patch, minor, or major. This will: change the version in `package.json`, commit that (locally), and make a tag in the format `vx.y.z`. Read more about that [on npm docs](https://docs.npmjs.com/getting-started/publishing-npm-packages#updating-the-package).
-+ Deploy npm package to the world; `npm publish`. (if you don't have [publish rights][uportal-app-framework access on npm] contact authors)
-+ Push git changes to the github via `git push origin master` and `git push origin <tag>`
+Before cutting the release, do what you can to square away the documentation for the release. Add the pull requests from the milestone to the [CHANGELOG](../CHANGELOG.md).
```

Comment:
I understand wanting a human touch in the README. :+1: :bust_in_silhouette: 
However if it is going to be *only* a commit list using [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog) or [changelog](https://www.npmjs.com/package/changelog) would be a good way to go. :robot:

Reply:
Good point. Since @vertein was a proponent of our current release process, this PR will wait at least a week until he gets back from vacation so he can have an opinion on this

Reply:
:+1: